app/view_models/admin/edit_auction_view_model.rb

```diff
@@ -14,24 +14,44 @@ def new_record?
   end
 
   def hour_default(field)
-    dc_time(field).strftime("%l").strip
+    if field_value(field).present?
+      dc_time(field).strftime("%l").strip
+    else
+      "1"
```

Comment:
I wouldn't sweat it too much though.

Sent from my iPhone

On May 27, 2016, at 6:20 PM, Jessie A. Young <notifications@github.com>
wrote:

In app/view_models/admin/edit_auction_view_model.rb
<https://github.com/18F/micropurchase/pull/620#discussion_r64973576>:

> @@ -14,24 +14,44 @@ def new_record?
>    end
>
>    def hour_default(field)
> -    dc_time(field).strftime("%l").strip
> +    if field_value(field).present?
> +      dc_time(field).strftime("%l").strip
> +    else
> +      "1"

yeah let me think about how we can make this less ugly

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/18F/micropurchase/pull/620/files/a0f13b309952c5e916ff6c0fa72877d75b258af9#r64973576>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAAH_DC0bof-J5z-cT8wA5FY_J41Ymhwks5qF25HgaJpZM4Io3WW>
.